lld/MinGW/Options.td

```diff
@@ -196,6 +196,9 @@ defm guard_longjmp : B<"guard-longjmp",
   "Do not enable Control Flow Guard long jump hardening">;
 defm error_limit:
   EqLong<"error-limit", "Maximum number of errors to emit before stopping (0 = no limit)">;
+def build_id: J<"build-id=">, HelpText<"Generate build ID note (pass none to disable)">, 
```

Comment:
What is one supposed to pass after `=` to enable it?

If we only support `-build-id` and `-build-id=none`, perhaps those should just be two flags instead?

Reply:
This is to be consistent with elf's build-id options, though it doesn't matter. The error message "unsupported build id hashing: {} using default hashing." is more user friendly than unknown flags if we handle it inside driver.